### PR TITLE
Feature/accessibility instruction

### PR DESCRIPTION
**Pull request by puranban**
### Changes

- Add accessibility modal feature instruction for the first time only

### NOTE
- User can dismiss the accessibility instructions modal from all types of project.

### CODE REVIEW

File: src/shared/common/AccessibilityInfoModal.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        getData();
+    }, []);
+
+    const closeModal = useCallback(() => {
+        try {
+            AsyncStorage.setItem('visitedRoute', 'true');
+            modalRef.current?.close();
+        } catch (e) {
+            console.log('error', e);
+        }
```
Comment (samshara) [inline src/shared/common/AccessibilityInfoModal.js:85]: maybe we can use `asyc` function here.

File: src/shared/common/AccessibilityInfoModal.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+import { COLOR_DEEP_BLUE } from '../constants';
+import Button from './Button';
+
+/* eslint-disable global-require */
+const GLOBAL = require('../Globals');
+
+const styles = StyleSheet.create({
+    modal: {
+        height: GLOBAL.SCREEN_HEIGHT < 500 ? GLOBAL.SCREEN_HEIGHT - 50 : 'auto',
```
Comment (tnagorra) [inline src/shared/common/AccessibilityInfoModal.js:15]: What is the logic behind this? What happens if height is zero or negative?

File: src/shared/common/AccessibilityInfoModal.js [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    },
+});
+
+function AccessibilityInfoModal() {
+    const modalRef = useRef();
+
+    const getData = async () => {
+        try {
+            //     Check if user has visited this route before
+            //     For simplicity, using a boolean flag stored in AsyncStorage
```
Comment (tnagorra) [inline src/shared/common/AccessibilityInfoModal.js:63]:
```suggestion
            // Check if user has visited this route before
            // For simplicity, using a boolean flag stored in AsyncStorage
```

File: src/shared/common/AccessibilityInfoModal.js [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+                <Text style={styles.text}>
+                    <Trans i18nKey="AccessibilityInstruction:tickIconInfo">
+                        Tap twice to turn the tile yellow and show a question
+                        icon
+                    </Trans>
+                </Text>
+            </View>
+            <View style={styles.row}>
+                <Image
+                    source={require('../views/assets/bad_image_new_icon.png')}
```
Comment (tnagorra) [inline src/shared/common/AccessibilityInfoModal.js:134]: Let's move these require at the top. Why are we not using import syntax?

File: src/shared/common/AccessibilityInfoModal.js [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+            </View>
+            <View style={styles.buttonContainer}>
+                <Button
+                    style={styles.closeButton}
+                    onPress={closeModal}
+                    textStyle={{
+                        fontSize: 13,
+                        color: '#ffffff',
+                        fontWeight: '700',
+                    }}
```
Comment (tnagorra) [inline src/shared/common/AccessibilityInfoModal.js:160]: Let's move this style to the styles

**Review by tnagorra — CHANGES_REQUESTED** [review]
Let's fix all of the issues before we can deploy the changes

File: src/shared/common/AccessibilityInfoModal.js (hunk 2 above)
Comment (puranban) [reply src/shared/common/AccessibilityInfoModal.js:13]: I don't think we need set modal  height according to the device, we can simply use `height: 'auto'` in this case.

@tnagorra what do you think?

Comment (tnagorra) [reply src/shared/common/AccessibilityInfoModal.js]: I don't have much idea about the usecase here.

**Review by tnagorra — APPROVED** [review]

**Pull request merged**